### DISCUSSION TITLE
garbage software

### DISCUSSION THREAD

**Discussion by Sequential-circuits** [nextcloud/recognize#1068] [General]
impossible to make it work with GPU: it insists in saying _Classifier process output: 2024-01-04 16:18:12.129427: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcusolver.so.11'; dlerror: libcusolver.so.11: cannot open shared object file: No such file or directory_

and yes I have cuda 11 and cuDNN SDK 8.1.0 installed and LD_LIBRARY set correctly to point to libcusolver.so.11

and it should be running on cuda 12 by now instead of cuda 11 which is obsolete, but apparently the author does not want to update it

and, finally, even if I uninstall tensorflow completely, it still shows that same error message

so, garbage software

**Comment by marcelklehr** [comment]
Hello 👋

Thank you for taking the time to open this discussion. Note also that GitHub is a place where people meet to make software better together. Nobody here is under any obligation to help you, solve your problems or deliver on any expectations or demands you may have. If enough people come together we can collaborate to make this software better. 
Unfortunately you have chosen a different path with this comment. If you feel, at any point, like you need to vent, this is not the place for it; you can go to the forum, to twitter or somewhere else. I have thus locked this conversation. In the future, make sure to focus on the tech and keep your opinions to yourself. (Also see our [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/). Really.)
